packages/gatsby-plugin-mdx/README.md

```diff
@@ -1,6 +1,6 @@
 ![Logo](../../img/gatsby-mdx.png)
```

Comment:
Is it possible to move the gatsby-mdx logo inside this package?

Reply:
Done!